ostree/ostree_stub.go

```diff
@@ -0,0 +1,3 @@
+// +build containers_image_ostree_stub
```

Comment:
Why is this necessary? AFAICS the `PACKAGES` definition already excludes `c/i/ostree` from the `Makefile` operations.

If an external user did refer to the `ostree` package while the stub tag is enabled, failing with `no buildable Go source files in …/github.com/containers/image/ostree` seems a bit better than failing with `undefined: ostree.SomeSymbol`.

Reply:
ok